### PR TITLE
docs: add installation videos

### PR DESCRIPTION
**Pull request by thesephirot**

### CODE REVIEW

File: docs/docs/guides/deploy/compose.mdx [hunk 1]
```diff
@@ -25,5 +25,8 @@ docker compose up --detach
 ```
 
 <DefaultUser components={props.components} />
+
+## VideoGuide
+<iframe width="560" height="315" src="https://www.youtube.com/embed/-02FaoN9Fko" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (eliobischof) [inline docs/docs/guides/deploy/compose.mdx:30]:
```suggestion
<iframe width="100%" height="315" src="https://www.youtube.com/embed/-02FaoN9Fko" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/-02FaoN9Fko" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: docs/docs/guides/deploy/knative.mdx [hunk 2]
```diff
@@ -59,5 +59,7 @@ If you didn't configure something else, this is the default IAM admin users logi
 * username: zitadel-admin@<span></span>zitadel.zitadel.default.127.0.0.1.sslip.io
 * password: Password1!
 
+## VideoGuide
+<iframe width="560" height="315" src="https://www.youtube.com/embed/m3TXmz3cK7E" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (eliobischof) [inline docs/docs/guides/deploy/knative.mdx:63]:
```suggestion
<iframe width="100%" height="315" src="https://www.youtube.com/embed/m3TXmz3cK7E" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/m3TXmz3cK7E" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: docs/docs/guides/deploy/linux.mdx [hunk 3]
```diff
@@ -37,5 +37,9 @@ ZITADEL_EXTERNALSECURE=false zitadel start-from-init --masterkey "MasterkeyNeeds
 ```
 
 <DefaultUser components={props.components} />
+
+## VideoGuide
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YVLua-q7dbs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (eliobischof) [inline docs/docs/guides/deploy/linux.mdx:42]:
```suggestion
<iframe width="100%" height="315" src="https://www.youtube.com/embed/YVLua-q7dbs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/YVLua-q7dbs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: docs/docs/guides/deploy/macos.mdx [hunk 4]
```diff
@@ -44,5 +44,10 @@ ZITADEL_EXTERNALSECURE=false zitadel start-from-init --masterkey "MasterkeyNeeds
 ```
 
 <DefaultUser components={props.components} />
+
+## VideoGuide
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Ob-kBzmezeg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (eliobischof) [inline docs/docs/guides/deploy/macos.mdx:49]:
```suggestion
<iframe width="100%" height="315" src="https://www.youtube.com/embed/Ob-kBzmezeg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/Ob-kBzmezeg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

**Review by eliobischof — CHANGES_REQUESTED** [review]

**Comment by thesephirot** [comment]
@eliobischof I am fine with the scaling.

**Review by eliobischof — APPROVED** [review]

**Pull request merged**